core/state_processor.go

```diff
@@ -237,18 +237,27 @@ func (p *StateProcessor) Process(block *types.WorkObject, etxSet *types.EtxSet)
 	startTimeSenders := time.Now()
 	senders := make(map[common.Hash]*common.InternalAddress) // temporary cache for senders of internal txs
```

Comment:
Your commit says "updated senders cache to LRU", but the senders cache is still a map. Did you forget to do the LRU bit?

Reply:
Additionally, there's a lot of changes in this commit that don't appear to have anything to do with the senders cache or LRUs vs maps. Could you explain what this commit is supposed to be doing?

Reply:
The senders cache is in the tx pool, not in the state processor. If you check the tx pool senders cache you'll see I changed it to be an LRU. What you have commented on here is a temporary map that gets filled with senders from the lookups. In addition, I disagree that changes in this commit don't have anything to do with the senders cache change. The only changes made that are not necessarily related to LRU but are related to the senders cache are ignoring the coinbase transaction in the lookup and adding Qi transactions to the senders cache (so musig is not done twice).